common/debug.go

```diff
@@ -0,0 +1,52 @@
+package common
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httputil"
+	"os"
+)
+
+func DebugRequest(req *http.Request, rsp *http.Response) {
```

Comment:
This might be more appropriately named as `PrintRequestAndResponse` or something like that, since that's a more literal representation of what it's doing.

Reply:
agreed, done